resilience4j-timelimiter/src/main/java/io/github/resilience4j/timelimiter/TimeLimiter.java

```diff
@@ -1,23 +1,31 @@
 package io.github.resilience4j.timelimiter;
 
+import io.github.resilience4j.core.EventConsumer;
+import io.github.resilience4j.timelimiter.event.TimeLimiterEvent;
+import io.github.resilience4j.timelimiter.event.TimeLimiterOnFailureEvent;
+import io.github.resilience4j.timelimiter.event.TimeLimiterOnSuccessEvent;
+import io.github.resilience4j.timelimiter.event.TimeLimiterOnTimeoutEvent;
 import io.github.resilience4j.timelimiter.internal.TimeLimiterImpl;
 
 import java.time.Duration;
-import java.util.concurrent.*;
+import java.util.concurrent.Callable;
+import java.util.concurrent.Future;
 import java.util.function.Supplier;
 
 /**
  * A TimeLimiter decorator stops execution after a configurable duration.
  */
 public interface TimeLimiter {
 
+    String DEFAULT_NAME = "TimeLimiter";
```

Comment:
or maybe better "UNDEFINED" or null?

Reply:
Yes, I agree. UNDEFINED might be better.